content/id/docs/reference/glossary/kube-apiserver.md

```diff
@@ -4,16 +4,15 @@ id: kube-apiserver
 date: 2019-04-21
 full_link: /docs/reference/generated/kube-apiserver/
 short_description: >
-  Komponen di master yang mengekspos API Kubernetes. Merupakan <i> front-end </i> dari <i> kontrol plane </i> Kubernetes.
+  Komponen _control plane_ yang mengekspos API Kubernetes. Merupakan _front-end_ dari _control plane_ Kubernetes.
 
 aka:
 tags:
 - architecture
 - fundamental
 ---
- Komponen di master yang mengekspos API Kubernetes. Merupakan <i> front-end </i> dari <i> kontrol plane </i> Kubernetes.
+Komponen _control plane_ yang mengekspos API Kubernetes. Merupakan _front-end_ dari _control plane_ Kubernetes.
 
 <!--more-->
 
-Komponen ini didesain agar dapat di-<i>scale</i> secara horizontal. Lihat [Membangun Klaster HA](/docs/admin/high-availability/).
-
+Komponen ini didesain agar dapat diskalakan secara horizontal. Lihat [Membangun Klaster HA](/docs/admin/high-availability/).
```

Comment:
“horizontal” is not standard.
ID link is available.
```suggestion
Komponen ini didesain agar dapat diskalakan secara horisontal. Lihat [Membangun Klaster HA](/id/docs/admin/high-availability/).
```

Reply:
It's the other way around... horisontal is the non official word, as described here https://kbbi.kemdikbud.go.id/entri/horizontal